scripts/ci/build-and-test.sh

```diff
@@ -10,20 +10,29 @@ cd $(dirname $0)/../..
 source scripts/ci/sources/mode.sh
 source scripts/ci/sources/tunnel.sh
 
+# Check if tests can be skipped
+skip_tests=true
+for filename in $(git diff --name-only $TRAVIS_BRANCH...HEAD); do
```

Comment:
https://github.com/travis-ci/travis-ci/issues/2668 ``TRAVIS_COMMIT_RANGE`` will include outdated commit if user does a force push (which is very common to updating a PR)